structure/0072-legal-team.md

```diff
@@ -0,0 +1,85 @@
+# RFC 0072: Legal Team
```

Comment:
On a point of formatting: all headers within this document other than this title header should be increased by one level, as there should only be one H1 header (this one).

Reply:
Hard disagree - it makes the file much more navigable by only going from h1 to h3. It doesn't make sense to only have h2.

Reply:
Can you please clarify for me what you mean? Are you saying that it is more navigable to have multiple H1 headers with a single nested level of H2 headers, compared to an H1 header for the title, H2 headers for each section, and a nested level of H3 headers (under the Explanation section)?

Based on a survey of all current RFCs, 9 out of 16 RFCs use H2 headers for the RFC sections, with 8 of those H2-using RFCs (and 1 of H1-using RFCs) using the H1 header as the title of the RFC.

If we expand the survey to all merged _and_ proposed RFCs, then 16 out of 26 use H2 headers, with 11 of these using H1 for the RFC title. The RFC template also uses H2 headers, with a single H1 header.

I therefore believe that, for better consistency with the majority of the current and proposed RFCs, the headings here should be modified so only one H1 header remains as the title, with sub-headings using H2 and lower-ranked headings.